src/pieces/commands/assets.py

```diff
@@ -81,96 +81,52 @@ def open_asset(**kwargs):
     print_model_details(asset_dict["name"],asset_dict["created_at"],asset_dict["updated_at"],asset_dict["type"],asset_dict["language"],filepath)
 
 def save_asset(**kwargs):
-    ### THIS DOES NOT CURRENTLY WORK ###
-    print("Not Currently Working")
-
-    if not commands_functions.current_asset:
-        open_asset()
-    else:
-        asset_to_update = commands_functions.current_asset.get('id')
-        # Pass asset and file name
-        update_asset(asset_to_update, commands_functions.application)
+    asset = extract_asset_info(get_asset_by_id(commands_functions.current_asset))
+    file_path = os.path.join(open_snippet_dir , f"{commands_functions.sanitize_filename(asset["name"])}{commands_functions.get_file_extension(asset["language"])}")
+    print(f"Saving {file_path} to {asset['name']} snippet with uuid {commands_functions.current_asset}")
+    
+    # Pass asset and file name
+    update_asset(file_path, commands_functions.current_asset)
+    print("saved successfully")
 
 # Probably needs renamed. This only currently edits the Asset's name
```

Comment:
I would probably just adjust this name to be specific as well

Reply:
ie edit name
edit classification

Reply:
sorry that was an old comment and I removed